### PR TITLE
feat: disable MainPageStory

### PR DESCRIPTION
**Pull request by cxw620**
修改点击首页头像为跳转到个人主页
Fix https://github.com/yujincheng08/BiliRoaming/issues/700

### CODE REVIEW

**Comment by cxw620** [comment]
7.22 版本测试通过, 达到预期效果.

File: app/src/main/java/me/iacn/biliroaming/hook/MainPageStoryHook.kt [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+                hookAfterMethod("getUrl") { param ->
+                    param.result?.callMethod("setValue", "bilibili://root?tab_name=我的")
+                }
+                hookAfterMethod("getUrlV2") { param ->
+                    param.result?.callMethod("setValue", "bilibili://root?tab_name=我的")
+                }
+            }
+        }
+    }
+}
```
Comment (yujincheng08) [inline app/src/main/java/me/iacn/biliroaming/hook/MainPageStoryHook.kt:20]: add tailing new line

File: app/src/main/java/me/iacn/biliroaming/hook/MainPageStoryHook.kt [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package me.iacn.biliroaming.hook
+
+import me.iacn.biliroaming.utils.*
+
+class MainPageStoryHook(classLoader: ClassLoader) : BaseHook(classLoader) {
+    override fun startHook() {
+        Log.d("startHook: MainPageStory")
+        val disableMainPageStory = sPrefs.getBoolean("disable_main_page_story", false)
+        if (disableMainPageStory) {
```
Comment (yujincheng08) [inline app/src/main/java/me/iacn/biliroaming/hook/MainPageStoryHook.kt:9]: inline this variable?

**Comment by cxw620** [comment]
修改完毕

**Review by yujincheng08 — APPROVED** [review]

**Pull request merged**